publicodes/core/test/mécanismes/résoudre-le-cycle.yaml

```diff
@@ -0,0 +1,67 @@
+fx: 
+x:
+  résoudre le cycle: oui
+  valeur: fx
+  exemples:
+    - nom: affine
+      situation: 
+        fx: 200 - x 
+      valeur attendue: 100
+    - nom: quadratique
+      situation: 
+        fx: 0.2 * x * x - 400 * x + 500
+      valeur attendue: 2003.743
```

Comment:
On peut mettre un lien vers https://www.wolframalpha.com/input/?i=x%3D0.2x%C2%B2-400x%2B500 pour justifier que c'est la bonne valeur attendue.

Ici il y a deux solutions, donc le test risque de "casser" si on change la logique d'itération (premier essai notamment). C'est pas super gênant mais le lien vers wolframalpha permettra de garder ça en tête aussi. Je suis curieux de savoir comment les tableurs gèrent les solutions multiples.